suite-native/accounts/src/components/AccountListItemBase.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { TouchableOpacity, TouchableOpacityProps } from 'react-native';
+
+import { Box, HStack, Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+export type AccountListItemBaseProps = {
+    icon: React.ReactNode;
+    title: React.ReactNode;
+    badges?: React.ReactNode;
```

Comment:
optionals at the end

Reply:
fixed... we should take a look if these things coudn't be easily automated in ESlint